manifests/0000_26_cloud-controller-manager-operator_16_credentialsrequest-gcp.yaml

```diff
@@ -15,10 +15,45 @@ spec:
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: GCPProviderSpec
-    predefinedRoles:
-      - roles/compute.instanceAdmin
-      - roles/compute.loadBalancerAdmin
-      - roles/iam.serviceAccountUser
+    permissions:
+    - "compute.addresses.create"
+    - "compute.addresses.delete"
+    - "compute.addresses.get"
+    - "compute.addresses.list"
+    - "compute.firewalls.create"
+    - "compute.firewalls.delete"
+    - "compute.firewalls.get"
+    - "compute.firewalls.update"
+    - "compute.forwardingRules.create"
+    - "compute.forwardingRules.delete"
+    - "compute.forwardingRules.get"
+    - "compute.healthChecks.create"
+    - "compute.healthChecks.delete"
+    - "compute.healthChecks.get"
+    - "compute.healthChecks.update"
+    - "compute.httpHealthChecks.create"
+    - "compute.httpHealthChecks.delete"
+    - "compute.httpHealthChecks.get"
+    - "compute.httpHealthChecks.update"
+    - "compute.instanceGroups.addInstances"
+    - "compute.instanceGroups.create"
+    - "compute.instanceGroups.delete"
+    - "compute.instanceGroups.get"
+    - "compute.instanceGroups.listInstances"
+    - "compute.instanceGroups.removeInstances"
```

Comment:
Don't we need `use`, `list` and `update` permissions?

Reply:
I don't think we need that or at least didn't find any reference of that service call anywhere.

Reply:
`compute.instanceGroup.List` is used just for tests in this case. `Update` and `Use` are not present.